CDP4WebServices.API/Modules/10-25/EngineeringModelApi.cs

```diff
@@ -439,6 +442,19 @@ protected override Response PostResponseData(dynamic routeParams)
                 var errorResponse = new JsonResponse($"exception:{ex.Message}", new DefaultJsonSerializer());
                 return errorResponse.WithStatusCode(HttpStatusCode.Forbidden);
             }
+            catch (BadRequestException ex)
+            {
+                if (transaction != null && !transaction.IsCompleted)
+                {
+                    transaction.Rollback();
```

Comment:
i get the impression that rollback is done twice, here and the place where the BadRequestException is being raised, should be in one place only